lib/ansible/module_utils/basic.py

```diff
@@ -92,6 +92,7 @@
 from collections import deque
 from collections import Mapping, MutableMapping, Sequence, MutableSequence, Set, MutableSet
 from itertools import repeat, chain
+from ansible.module_utils.ec2 import camel_dict_to_snake_dict
```

Comment:
Is this needed in basic.py?  It adds another file that would need to be sent for every module if so.